### PR TITLE
fixing some shootOut problems

### PR DESCRIPTION
**Pull request by rolfvdhulst**
Fixes some shootOut Problems found during testing.

### CODE REVIEW

File: roboteam_ai/src/skills/KickTo.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#include <roboteam_ai/src/control/ControlUtils.h>
+
+namespace rtt {
+namespace ai {
+
+KickTo::KickTo(string name, bt::Blackboard::Ptr blackboard)
+        :Skill(std::move(name), std::move(blackboard)) {
+}
+void KickTo::onInitialize() {
+    shootPos=properties->getVector2("shootPos");
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/skills/KickTo.cpp]: never initialize an absolute position in the field.

this changes for fieldsize

File: roboteam_ai/src/skills/KickTo.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+/// Get an update on the skill
+bt::Node::Status KickTo::onUpdate() {
+    if (! robot) return Status::Running;
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/skills/KickTo.cpp]: not needed. and it should not return running

Comment (ThijsLuttikhuis) [reply roboteam_ai/src/skills/KickTo.cpp]: this is done in skill.cpp

File: roboteam_ai/src/skills/KickTo.cpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    if (world::field->pointIsInDefenceArea(ball->pos, false)) {
+        command.w = 0;
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/skills/KickTo.cpp]: why 0?

Comment (rolfvdhulst) [reply roboteam_ai/src/skills/KickTo.cpp]: it defaults to that anyways.

**Pull request merged**